packages/nextjs/contracts/predeployedContracts.ts

```diff
@@ -212,6 +212,7 @@ const preDeployedContracts = {
           ],
         },
       ],
+      classHash: universalEthAddress, //used to for demostration and to remove ts-error
```

Comment:
the class hash of ETH in devnet is : 0x046ded64ae2dead6448e247234bab192a9c483644395b66f2155f2614e5804b0

in sepolia : 0x07f3777c99f3700505ea966676aac4a0d692c2a9f5e667f4c606b51ca1dd3420

Reply:
@Jemiiah any updates on this ? you will have to write the class hash i provided here in this line. not overwrite the universalEthAddress with the values i provided above.